docs/website/docs/customization/customize-existing-pipeline/_category_.json

```diff
@@ -0,0 +1,7 @@
+{
+    "label": "Customize an existing pipeline",
```

Comment:
@adrianbr please use the new way of creating categories in `website/sidebars.js` instead of this method